gradio/helpers.py

```diff
@@ -767,6 +769,7 @@ def make_waveform(
         bars_color: Color of waveform bars. Can be a single color or a tuple of (start_color, end_color) of gradient
         bar_count: Number of bars in waveform
         bar_width: Width of bars in waveform. 1 represents full width, 0.5 represents half width, etc.
+        animate_waveform: Whether to animate the waveform
```

Comment:
Can we be more specific on what animate means here?
```suggestion
        animate_waveform: Whether to animate the waveform
```